### PR TITLE
Use folderid for persona replies

### PR DESCRIPTION
**Pull request by maxxer**

### CODE REVIEW

**Comment by maxxer** [comment]
It's not *instead*, it's using both conditions for persona replies

File: bin/subzim [hunk 1]
```diff
@@ -66,37 +66,43 @@ PERMISSIONS="rwixd"
...
+# Persona creation instructions
+BOXPFILE="$(mktemp /tmp/subzim-boxp.XXXXXXXX.txt)"
 
 echo "" > "${BOXMFILE}"
 echo "" > "${BOXSFILE}"
+echo "" > "${BOXPFILE}"
 echo "" > "${PROFFILE}"
 
 echo "mfg / account $SUBSCRIBER $PERMISSIONS" >> "${BOXMFILE}"
-echo "mfg /Inbox account $SUBSCRIBER $PERMISSIONS" >> "${BOXMFILE}"
```
Comment (barrydegraaff) [inline bin/subzim:78]: @maxxer can you please re-add these 4 lines, I know they seem redundant, but Zimbra sometimes does not share all folders when calling only the command with `/` as argument. Having these lines saves us a lot of issues reports on Github, thanks.

Comment (maxxer) [reply bin/subzim]: ok

**Comment by barrydegraaff** [comment]
Thanks, merged!

**Pull request merged**